portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -51,20 +51,24 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    if (request.getParameter(COMMENT_FORM) != null && !request.getParameter(COMMENT_FORM).isEmpty()) {
+    if (request.getParameter(this.COMMENT_FORM) != null && !request.getParameter(this.COMMENT_FORM).isEmpty()) {
       storeEntities(createEntities(request));
-    } else if (request.getParameter(MAX_FORM) != null && !request.getParameter(MAX_FORM).isEmpty()) {
-      setMaxComment(request, request.getParameter(MAXCOMMENT_PARAMETER));
+    } else if (request.getParameter(this.MAX_FORM) != null && !request.getParameter(this.MAX_FORM).isEmpty()) {
+      setMaxComment(request, request.getParameter(this.MAXCOMMENT_PARAMETER));
     }
 
-    response.sendRedirect(COMMENT_HTML_PARAMETER);
+    response.sendRedirect(this.COMMENT_HTML_PARAMETER);
   }
 
   public Entity createEntities(HttpServletRequest request){
-    Entity commentEntity = new Entity(COMMENT_PARAMETER);
-    commentEntity.setProperty(USERNAME_PARAMETER, request.getParameter(this.USERNAME_PARAMETER));
-    commentEntity.setProperty(REFLECTION_PARAMETER, request.getParameter(this.REFLECTION_PARAMETER));
-    commentEntity.setProperty(DATE_TIME_PARAMETER, DATE_TIME_FORMATTER.format(LocalDateTime.now()).toString());
+    Entity commentEntity = new Entity(this.COMMENT_PARAMETER);
+    System.out.println(request.getParameter(this.USERNAME_PARAMETER));
+    System.out.println(request.getParameter(this.REFLECTION_PARAMETER));
+    System.out.println(DATE_TIME_FORMATTER.format(LocalDateTime.now()).toString());
```

Comment:
`this.DAT_TIME_FORMATTER`

Reply:
This is fixed as well in the many_dev branch